encryption/hqc/hqc/impl/aarch64.yaml

```diff
@@ -0,0 +1,6 @@
+name: aarch64
+sources:
+  - 'https://bench.cr.yp.to/results-kem.html'
+platform: 'aarch64, rpi4ubuntu64, armv8, 4 x 1500MHz'
+hardware features:
+  - '4 x 1500MHz, A72 (410fd083); 2019 Broadcom BCM2711'
```

Comment:
please note that `impl` and `bench` are different things.

`implementation` is just about static code, things like cpu speed are properties of the `benchmark`

`impl.platform` is just something like `c`, `assembler` or `fpga` (I agree the name clash is unfortunate).

about the hardware features: see above